compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/internal/parser/BallerinaParser.java

```diff
@@ -4936,13 +4936,15 @@ private boolean isEndOfExpression(SyntaxKind tokenKind, boolean isRhsExpr, boole
         }
     }
 
+    private STNode parseBasicLiteral() {
+        return parseBasicLiteral(consume());
```

Comment:
Having a method for a single line of code sounds not correct. It also adds a runtime over-head of an additional function call.  
Since this is a most prominent usage, shall we keep this as is, and only change the place where the change is needed?